rfcs/text/0000-planning-for-the-dcp.md

```diff
@@ -0,0 +1,209 @@
+
+### DCP Product Backlog
+
+The DCP ZenHub board is the ordered **!** **Scrum** Product Backlog for DCP. 
+
+The top-level *dcp* repo MUST contain **all** significant features that are important to the DCP community as a whole and require visibility and prioritization. This includes  issues or epics describing user features or product-wide technical requirements (e.g. “components should have logs...“). 
+
+All component repos that will assign items to a DCP Milestone MUST be added to the ZenHub *DCP Backlogs* workspace.
+
+DCP Product Backlog items  are modeled as:
+- [User Stories](http://agiledictionary.com/277/user-story/) with acceptance criteria
+- [Epics](http://agiledictionary.com/309/epic/) contain a related set of smaller User Stories required to complete an objective.
+- [Spikes](http://agiledictionary.com/209/spike/) adopted from eXtreme Programming to answer design questions.
+  - [Example use in the Scaled Agile Framework](https://www.scaledagileframework.com/spikes/)
+
+#### User Stories
+
+The [*researcher with a keyboard*](https://docs.google.com/drawings/d/1Li9o_uVkEZDZHbkQv86I49FG8I0zGSgHYkKabp7piEs/edit) and [*researcher with a pipette*](https://docs.google.com/drawings/d/1o6I5BLtJuRJ1vUdb7dHCkFbo0XiaxKDBiVAogqnUowU/edit) journeys are the initial source of user stories that  are modeled as epics in the *dcp* repo,  reflecting the narrative flow of the product. Further refinement by the Product Owners is required to both develop and  extend the high-level set of user activities. 
+
+Since journeys may vary per persona, personas MUST be identified for an epic by assigning a documented GitHub label such as "keyboard", "pipette", or "road-builder".  An epic MAY support multiple personas.
+
+#### Technical and Operational Requirements
+Product-wide technical or operational requirements are defined by **Technical Architecture** and modeled as epics in the *dcp* repo.
+
+#### Owners
+
+A DCP owner MUST be assigned to each issue or epic in the *Product Backlog* in the *dcp* repo to shepherd and coordinate progress. This owner is responsible for ensuring that related issues are created in component repo(s) and linked to epics in the *dcp* repo.
+
+For example, an implementation task for a specific component resides in its own ZenHub repo and is linked to the relevant epic in the *dcp* repo.
+
+* No component-specific GitHub labels such as "Data Store" MUST be assigned to an epic in the *dcp* repo. The link documents the relationship.
+
+### Sprints
+
+ **Four week sprints**. There MUST be one DCP Sprint and Sprint Backlog, not a separate Sprint and Sprint Backlog for each component team. Each DCP Sprint starts on Thursday and ends on Wednesday.  
+ 
+Each Sprint MUST be assigned a ZenHub *Milestone* with a due date by the **Product Owners**.
+
+ All Scrum events are scheduled on the DCP calendar at a regular cadence. For example:
+![](../images/0000-scrum-calendar.png)
+
+### Refinement
```

Comment:
This section was very hard for me to parse. I'd hit some very basic points in the first couple sentences and then unroll notes, questions, and details after the fact:

1. What are the goals of a refinement meeting?
1. Who attends?
1. How often?
1. Details, FAQ
1. Supporting historical info, notes

Reply:
@mweiden - I restructured this section as a first pass. I plan to review further, but please let me now if it is easier to _parse_.